src/main/java/uk/gov/mca/beacons/api/controllers/OwnerController.java

```diff
@@ -1,35 +1,44 @@
 package uk.gov.mca.beacons.api.controllers;
 
 import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.UUID;
 import javax.validation.Valid;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
+import uk.gov.mca.beacons.api.domain.PersonType;
 import uk.gov.mca.beacons.api.dto.BeaconPersonDTO;
 import uk.gov.mca.beacons.api.dto.WrapperDTO;
+import uk.gov.mca.beacons.api.exceptions.ResourceNotFoundException;
 import uk.gov.mca.beacons.api.jpa.entities.Person;
 import uk.gov.mca.beacons.api.mappers.BeaconPersonMapper;
 import uk.gov.mca.beacons.api.services.CreateOwnerService;
+import uk.gov.mca.beacons.api.services.GetPersonByIdService;
 
 @RestController
-@RequestMapping("/owner")
+@RequestMapping("/owners")
```

Comment:
For consistency with other URIs and to match the examples at jsonapi.org, changed the URI resource to be plural.  Have checked the Webapp repo and this shouldn't be a breaking change as this endpoint isn't used yet.